lib/beaker/dsl/helpers/host_helpers.rb

```diff
@@ -394,6 +394,16 @@ def install_package host, package_name, package_version = nil
           host.install_package package_name, '', package_version
         end
 
+        # Uninstall a package on a host
+        #
+        # @param [Host] host             A host object
+        # @param [String] package_name   Name of the package to uninstall
+        #
+        # @return [Result]   An object representing the outcome of *uninstall command*.
+        def uninstall_package host, package_name
```

Comment:
@Dakta I'm curious as to why you've added a host helper method here, I thought that you were in the camp that these methods shouldn't exist, we should be using host methods themselves rather than having these DSL proxy methods?

Reply:
I may not like them, but at least they should be consistent. I can't very well deprecate `#install_package`, and it felt awkward to not have it match. So I chose to write a matching one-line wrapper method. It'll be just as easy to decommission as `#install_package` since they both operate on a single host.